packages/lit-starter-js/package.json

```diff
@@ -39,34 +39,27 @@
   "author": "Google LLC",
   "license": "BSD-3-Clause",
   "dependencies": {
-    "lit": "^2.1.0"
+    "lit": "^2.2.4"
   },
   "devDependencies": {
-    "@11ty/eleventy": "^0.12.1",
-    "@11ty/eleventy-plugin-syntaxhighlight": "^3.0.1",
+    "@11ty/eleventy": "^1.0.1",
+    "@11ty/eleventy-plugin-syntaxhighlight": "^4.0.0",
     "@babel/eslint-parser": "^7.17.0",
     "@custom-elements-manifest/analyzer": "^0.5.7",
-    "@esm-bundle/chai": "^4.1.5",
-    "@open-wc/testing": "^3.0.0-next.1",
-    "@open-wc/testing-karma": "^4.0.9",
-    "@rollup/plugin-node-resolve": "^13.0.4",
-    "@rollup/plugin-replace": "^3.0.0",
-    "@types/mocha": "^9.0.0",
-    "@web/dev-server": "^0.1.22",
-    "@web/dev-server-legacy": "^0.1.4",
-    "@web/dev-server-rollup": "^0.3.9",
-    "@web/test-runner": "^0.13.16",
-    "@web/test-runner-mocha": "^0.7.4",
-    "@web/test-runner-playwright": "^0.8.4",
+    "@open-wc/testing": "^3.1.5",
```

Comment:
So I guess the Chai globals are coming in via this package?

Reply:
Yes, `@open-wc/testing` has `@esm-bundle/chai` as a dependency and re-exports so we don't need to depend on it ourselves.